sdk/communication/communication-short-codes/test/public/utils/testUSProgramBrief.ts

```diff
@@ -2,20 +2,28 @@
 // Licensed under the MIT license.
 
 import { RestError } from "@azure/core-rest-pipeline";
-import { ShortCodesClient } from "../../../src";
-import { USProgramBrief } from "../../../src";
-import { assert } from "chai";
 import {
-  CompanyInformation,
-  MessageDetails,
-  ProgramDetails,
-  TrafficDetails,
-} from "../../../src/generated/src/models/mappers";
+  CompanyInformationMapper,
+  MessageDetailsMapper,
+  ProgramDetailsMapper,
+  ShortCodesClient,
+  TrafficDetailsMapper,
+  USProgramBrief,
+} from "../../../src";
+import { assert } from "chai";
 import { CompositeMapper } from "@azure/core-client";
+import { isPlaybackMode } from "@azure-tools/test-recorder";
+import { v1 as uuid } from "uuid";
 
 export function getTestUSProgramBrief(): USProgramBrief {
+  let programBriefId = uuid();
+
+  if (isPlaybackMode()) {
+    programBriefId = "9d787bd6-07fc-4c7b-8e57-17f1fee41298";
+  }
```

Comment:
Instead of reassigning the value if is playback mode, I would suggest refactoring, so it is a constant value that is only assigned once.
```suggestion
  const programBriefId = isPlaybackMode() ? "9d787bd6-07fc-4c7b-8e57-17f1fee41298" : uuid();
```